README.md

```diff
@@ -293,3 +279,60 @@ func startOrchextraContentManager() {
 	}
 }
 ```
+
+### Third-party providers
+
+OCM provides a way to configure some Third-party providers by sending some information to the SDK. All providers can be configured in:
+
+```swift
+OCM.shared.providers = ...
+```
+
+The list of supported providers is:
+
+#### VIMEO
+
+1. Information needed by the SDK
+	* **Access token**
+2. How to configure in SDK:
+
+```swift
+let ocmProviders = Providers()
+ocmProviders.vimeo = VimeoProvider(accessToken: "xxxxxxx")
+OCM.shared.providers = ocmProviders
+```
+
+
+### Events information
+
+OCM informs about some events that could be of your interest (i.e. for analytics). There is a protocol calls OCMEventDelegate that contains some methods with relevant information about the OCM behaviour. To enable it you can set:
```

Comment:
I'd suggest to update it to something like this:
To be informed about different events of interest occurring on OCM (e.g: content being loaded, content being shared, etc.) that could come in handy for handling analytics events, you'll need to conform to the protocol OCMEventDelegate. You can set this delegate as follows: